src/assimp/assimp.cpp

```diff
@@ -506,7 +508,13 @@ assimp::Implementation::BindState assimp::Implementation::processMaterials(const
             material->Get(AI_MATKEY_COLOR_EMISSIVE, pbr.emissiveFactor);
             material->Get(AI_MATKEY_GLTF_ALPHACUTOFF, pbr.alphaMaskCutoff);
 
-            if (material->Get(AI_MATKEY_TWOSIDED, isTwoSided); isTwoSided)
+            bool optionFlag{false};
+            if(options->getValue("two_sided", optionFlag) && optionFlag) 
```

Comment:
"two_sided" should be replaced by assimp::two_sided so that if the string in the header is changed then everything will remain consistent.

Reply:
Ah, just saw this. Will change it.